.travis.yml

```diff
@@ -27,10 +27,15 @@ install: mvn install -q -DskipTests=true
 
 script: mvn test -q
 
+# Bucket-type to activate before running Riak KV tests.
```

Comment:
Can the Riak test class create these if they don't exist? Instead of doing it outside of the tests and depending on changing travis.yml?